### PR TITLE
Tasks

### PR DESCRIPTION
**Pull request by lewisevan**
Adds a new feature that allows admins to create and assign tasks. Members can view their assigned tasks from the homepage, and mark the tasks as started/completed.

### CODE REVIEW

**Comment by JettBurns14** [comment]
Cool, thanks for the edits! I'll check this out locally and merge if things look good!

File: public/js/admin.js [hunk 1]
```diff
@@ -119,6 +119,59 @@ let editUser = (e) => {
     });
 }
 
+let addTask = (e) => {
+    e.preventDefault();
+
+    let body = {};
+    for (key of e.target) {
+        if (key.name === "edit_contest_current") {
```
Comment (JettBurns14) [inline public/js/admin.js]: Note to self: rename this key.

**Comment by lewisevan** [comment]
You can just go ahead and re-commit the changes in another branch. That will probably be the easiest way to merge them.

Your changes look great by the way! Except now I've lost the ability to edit announcement messages....

**Comment by JettBurns14** [comment]
Yeah, will do!

Huh, everything should be working... I'll check on it after this merge.

File: views/pages/admin.ejs [hunk 2]
```diff
@@ -274,6 +275,14 @@
...
                                 <%= c.evaluator_kaid %>
                             </p>
+                            <p>
+                                <span class="bold">Username:</span>
+                                <%= c.username %>
+                            </p>
+                            <p>
+                                <span class="bold">Email:</span>
+                                <%= c.email %>
+                            </p>
```
Comment (JettBurns14) [inline views/pages/admin.ejs]: Note to self: Add this to my own branch later after merge.

**Pull request closed without merging**